### ISSUE TITLE
Possibility of a ReAgnus for Amiga 1000

### ISSUE THREAD

**Issue by markusbkk** [nonarkitten/amiga_replacement_project#17]
I'm curious. I know the chances of this are slim and the architecture of the Amiga 1000 is unique in some sense (it's the only OCS model without a Gary), but is there even the slightest possibility of making ReAgnus (Willoe) work with an Amiga 1000, unlocking 2MB of ChipMem?

I found this project earlier, so the form factor shouldn't be the main concern

https://github.com/LinuxJedi/A500-2Meg-Chip-ram

**Comment by nonarkitten** [comment]
Yes, it's possible to have 2MB (or more) on a Skinny Agnus, but the CPU is only able to see 512MB regardless due to the bus design of the Amiga 1000.

**Comment by markusbkk** [comment]
> Yes, it's possible to have 2MB (or more) on a Skinny Agnus, but the CPU is only able to see 512MB regardless due to the bus design of the Amiga 1000.

I understand. I was just wondering if there's a way to work around it somehow. 
I figured since Willoe is going to overcome some limitations in later models and feature up to 8MB of chipmem on those newer models, something like 2MB chipmem Amiga 1000 would be just as likely.

On NTSC models of the A1000 it's already [possible](https://www.amigalove.com/viewtopic.php?t=749) to get 2MB of chipmem, through a modified Rejuvenator. Unfortunately, my model is a European one without a daughterboard.

**Comment by nonarkitten** [comment]
Well a possible solution might be to add two alligator clips to the missing address lines.

**Comment by celso** [comment]
I too have a PAL A1000 board and would love to see this happening. If all it'd take to get the CPU to see a 2MB+ Skinny Agnus is connecting the missing address lines in the bus, then I'd totally go for it. It'd be a much better alternative to PAL owners than having to switch to a NTSC board + Rejuvenator.
